core/src/main/java/hudson/model/ParameterDefinition.java

```diff
@@ -152,6 +156,7 @@ public String getFormattedDescription() {
     }
 
     @Override
+    @Nonnull
```

Comment:
the same as before

Reply:
The descriptor should be nonnull since we use getDescriptorOrDie